database-migration/src/main/java/uk/ac/ebi/eva/mongo/migration/MongoMigrationMain.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2016 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.mongo.migration;
+
+import com.github.mongobee.Mongobee;
+import com.github.mongobee.changeset.ChangeLog;
+import com.github.mongobee.changeset.ChangeSet;
+import com.github.mongobee.exception.MongobeeException;
+import com.mongodb.BasicDBObject;
+import com.mongodb.BulkWriteOperation;
+import com.mongodb.DBCollection;
+import com.mongodb.DBCursor;
+import com.mongodb.DBObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.data.mongodb.core.MongoTemplate;
+
+import java.util.Objects;
+
+/**
+ * Script that executes the following steps using mongobee:
+ *  - Extracts the 'st' field from a Variant stored in MongoDB into a new statistics object
+ *  - _id, chr, start, end, ref and alt are added into the Statistics object
+ *  - The Statistics object is added into the statistics collection
+ *  - The 'maf' field is added into the Variant object
+ *  - The 'st' field is removed from the Variant object
+ *
+ * https://github.com/mongobee/mongobee/wiki/How-to-use-mongobee
+ * https://gist.github.com/jmmut/8439d830dac1954e93f9697051bf5a69
+ *
+ * Mongo URI example
+ * local: 127.0.0.1:27017
+ * remote: username:password@host:27017/admin
+ *
+ * ================
+ * Usage:
+ * java -jar database-migration-0.1-jar-with-dependencies.jar database_name collection_name mongo_uri
+ *
+ */
+@ChangeLog
+public class MongoMigrationMain {
+    private static final Logger logger = LoggerFactory.getLogger(MongoMigrationMain.class);
+
+    private static String dbName;
+    private static String variantsCollectionName;
+    private static String mongoUri;
+
+    public static void main(String[] args) throws MongobeeException {
```

Comment:
Can we extract the logic from this method into a class? There will be at least 2 other migrations in the future, one for the annotation and another for the files, and this would reduce the amount of refactoring we need to do.

Some of the variables declared above can be probably shared among all the Mongo migration scripts, but the collections are specific to a changelog.